### PR TITLE
turbine: Better auto-detection of R location

### PR DESCRIPTION
**Pull request by jmjwozniak**

### CODE REVIEW

**Comment by jmjwozniak** [comment]
Is this ok to merge now?

File: var/spack/repos/builtin/packages/turbine/package.py [hunk 1]
```diff
@@ -71,6 +73,13 @@ def configure_args(self):
             args.append('--with-python-exe={0}'.format(
                         self.spec['python'].command.path))
         if '+r' in self.spec:
-            args.append('--with-r={0}/rlib/R'.format(
-                        self.spec['r'].prefix))
+            r_location = '{0}/rlib/R'.format(self.spec['r'].prefix)
+            if not os.path.exists(r_location):
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/turbine/package.py]: @jmjwozniak this appears to be looking for R in the case that the prefix does not exist, which should be ensured by Spack. Are you working on a system with an external R (i.e. one registered in `packages.yaml`) that is provided by a module and has no discernible prefix?

Comment (scheibelp) [reply var/spack/repos/builtin/packages/turbine/package.py]: > this appears to be looking for R in the case that the prefix does not exist, which should be ensured by Spack

I should say that it is *usually* the case in Spack; although right after I mention a valid scenario where that would not be the case.

Mainly I'm curious what conditions are making this necessary for you.

Comment (jozik) [reply var/spack/repos/builtin/packages/turbine/package.py]: Yes, that's exactly the use case we're targeting (an external R registered in packages.yaml).

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**